### ISSUE TITLE
Docs issue: "Install Vector using YUM" is missing a pipe character

### ISSUE THREAD

**Issue by wimax-grapl** [timberio/vector#8385]
Problematic docs page: https://vector.dev/docs/setup/installation/package-managers/yum/

What it says:
```
curl -1sLf 'https://repositories.timber.io/public/vector/cfg/setup/bash.rpm.sh' \
  sudo -E bash
```
(This actually silently fails and doesn't execute anything 😱)

What it should say:
```
curl -1sLf 'https://repositories.timber.io/public/vector/cfg/setup/bash.rpm.sh' \
 | sudo -E bash
# note the pipe
```

Compare with the instructions for apt:
```
curl -1sLf \
  'https://repositories.timber.io/public/vector/cfg/setup/bash.deb.sh' \
| sudo -E bash
```

**Comment by jszwedko** [comment]
It sure is! Thanks for the report @wimax-grapl . I opened #8386 to address.

**Comment by jszwedko** [comment]
Closed by #8386
